### ISSUE TITLE
Default class/function display settings

### ISSUE THREAD

**Issue by rkucmierowski** [oleg-shilo/codemap.vscode#94]
Hi,
Is it possible to define a default state for displaying class/function elements or at least keep the last used setting? 
Now, when I reopen the file, both types (class/function) are displayed, and I prefer to display only the class map.
Kind regards

**Comment by oleg-shilo** [comment]
Yes it is. The "CODEMAP SETTING" panel is specifically designed for that:

![image](https://github.com/oleg-shilo/codemap.vscode/assets/16729806/34547f79-75a4-4617-a8df-a55c920e141b)

It was a contribution of one of the CodeMap users: @Timhobson-source. Probably he experienced the same UX challenges.

**Comment by rkucmierowski** [comment]
Ok, thanks, but when I close the project and reopen it, these settings reset and I have to select each time which options to display.

**Comment by oleg-shilo** [comment]
Ah... then we can make it persistent between the sessions.
Changing it to the enhancement. Will implement it over weekend.

**Comment by oleg-shilo** [comment]
Done.
Please update the extension to the v1.23.0
